nginx/device.conf

```diff
@@ -0,0 +1,7 @@
+# taken from https://gist.github.com/perusio/1326701
+
+map $http_user_agent $ua_device {
+	default 'pc';
```

Comment:
Can we use 'desktop' instead of 'pc' as default? Sounds better, is used in the referenced example too and will include Mac users. ;)

Reply:
Sure, added with breaking change comment. 👍